tests/reactable_test.jsx

```diff
@@ -1582,7 +1619,7 @@ describe('Reactable', function() {
                 ReactableTestUtils.expectRowText(2, ['Third']);
             });
         });
-    
+
```

Comment:
Let's keep this blank spaces at end of line untouched for the sake of a concise diff.